.eslintignore

```diff
@@ -4,3 +4,4 @@ lib
 node_modules
 standalone
 templates
+/src/frameworks/docs/**
```

Comment:
Put in `.eslintrc.js` (`ignorePatterns`) instead

Reply:
Oh -- thanks!!